tests/CacheCompat/CommonCache.Test.MsalJava/JavaLanguageExecutor.cs

```diff
@@ -33,7 +33,7 @@ public JavaLanguageExecutor(string className)
 
             try
             {
-                string compileArguments = $"-f \"{pomFilePath}\" compile";
+                string compileArguments = $"-f \"{pomFilePath}\" compile versions:use-latest-releases";
```

Comment:
Looks like this can be used to pick up the latest dependencies. It forces ALL dependecies to latest ver, but this project has very few so it's ok. I don't know how to make it use only MSAL latest.

Reply:
What needs to be ensured is that we use the latest MSAL4J, dependencies not so much, we should just use what ever the Java team tested with imo

---
In reply to: [460200295](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1959#discussion_r460200295) [](ancestors = 460200295)

Reply:
in order to pick up latest version of only one dependency "versions:update-properties" can be used -> "versions:update-properties updates properties defined in a project so that they correspond to the latest available version of specific dependencies. This can be useful if a suite of dependencies must all be locked to one version."

Reply:
also versions-maven-plugin should be added to pom.xml to use "versions:"

Reply:
Will go with Avery's suggestion and use version interval.